Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -920,3 +926,48 @@ lemma mulEquivOfOrderOfEq_symm_apply_gen : (mulEquivOfOrderOfEq hg hg' h).symm g
 end mulEquiv
 
 end generator
+
+section prod
+
+/-- The product of two finite groups is cyclic iff
+both of them are cyclic and their orders are coprime. -/
+@[to_additive "The product of two finite groups is cyclic iff
+both of them are cyclic and their orders are coprime."]
```

Comment:
Did you intend for the text here to be identical to the multiplicative version above it?